<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BigFileHardLinkClass</class>
 <widget class="QMainWindow" name="BigFileHardLinkClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>重复大文件创建硬链接</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>盘符</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTableView" name="tableDrive"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>文件大小</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinFileSize">
        <property name="suffix">
         <string>MB</string>
        </property>
        <property name="prefix">
         <string>&gt;</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>线程数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinThreadCount">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>文件名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editFileNames">
        <property name="text">
         <string>*.*</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnScan">
        <property name="text">
         <string>开始扫描</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>结果</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTreeView" name="treeFile"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>扫描结果：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelResult">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnCreateHardlink">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>创建硬链接</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>734</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BigFileHardLink.qrc"/>
 </resources>
 <connections/>
</ui>
